### PR TITLE
Jlechowicz/333 ui updates

### PR DESCRIPTION
**Pull request by jlechowicz**
This PR contains fixes for the accessibility issues Tina outlined in #333 

===

Resolves #XXX

- [X] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

**Comment by nguyen-tina** [comment]
>>>EXCITING STUFF HAPPENING IN THIS TICKET for those who like alignment 😌  <<<

File: src/client/App.scss [hunk 1]
```diff
@@ -128,17 +128,53 @@ footer {
 // radio question was answered for the feedback to show.
 .was-validated .unchecked .invalid-feedback {
   display: block;
+  color:  #CC2435;
```
Comment (sharonwarner) [inline src/client/App.scss:131]: Awesome, thanks!

**Comment by jlechowicz** [comment]
> Ah, this looks so much better! Do you mind just confirming that it looks good on both tablets and a mobile device?

Yep, looks good in both. Screenshots for @mkhandekar coming shortly

**Comment by jlechowicz** [comment]
image
image
image
image
image
@mkhandekar Screenshots aplenty! :)

**Comment by mthong** [comment]
Looks good to me.

**Pull request merged**